### PR TITLE
Refactor component to use new token structure

### PR DESCRIPTION
**Pull request by cmarin001**
This pull request encompasses a comprehensive refactor of a `<CountdownBar>`  within our application to align it with a newly introduced token structure. This strategic update is aimed at enhancing the component's design consistency, scalability, and maintainability by leveraging the advanced capabilities provided by the new design tokens.

### CODE REVIEW

File: src/CountdownBar/CountdownBar.stories.tsx [hunk 1]
```diff
@@ -1,8 +1,6 @@
-import { ThemeProvider } from "styled-components";
 import { CountdownBar, ICountdownBarProps } from ".";
```
Comment (JanuarAsprilla) [inline src/CountdownBar/CountdownBar.stories.tsx:1]: change `ICountdownBarProps` to `ICountdownBar`.
make this change wherever that interface is located

Comment (cmarin001) [reply src/CountdownBar/CountdownBar.stories.tsx]: solved

**Review by JanuarAsprilla — APPROVED** [review]

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/CountdownBar/props.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { inube } from "@inubekit/foundations";
 
-export type Appearance = keyof typeof inube.color.stroke;
+const appereances = Object.keys(inube.countdownBar);
+type Appearance = typeof appereances;
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/CountdownBar/props.ts:11]: solved

[the remark this answers is not in the document]
Comment (wfercanas) [reply src/CountdownBar/props.ts:11]: You're missing `(typeof appearances)[number]`

**Review by wfercanas — CHANGES_REQUESTED** [review]

**Review by wfercanas — APPROVED** [review]

**Pull request merged**